lib/managers/live_view_manager.dart

```diff
@@ -76,10 +80,19 @@ abstract class LiveViewManagerBase with Store, UiLoggy {
             return;
           }
 
+          Lock frameOrderLock = Lock();
+
           LiveViewManagerBase.instance._liveViewSubscription = stream.getStream().listen((frame) async {
             // New frame arrived
-            LiveViewManagerBase.instance.lastFrameImage = await frame.toImage();
-            LiveViewManagerBase.instance._liveViewState = LiveViewState.streaming;
+            if (LiveViewManagerBase.instance._timeFromLastReceivedFrame.elapsedMilliseconds < LiveViewManagerBase._minimumFrameTimeMs) {
+              StatsManagerBase.instance.addLiveViewFrameDroppedByConsumer();
```

Comment:
So it drops frames that come in too fast to make sure a queue doesn't form right?

Reply:
Correct ✅ this might solve the frame rush in a few cases when frames have been building up